Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeAdPermissions.ps1

```diff
@@ -100,68 +101,74 @@ Function Get-ExchangeAdPermissions {
         $domainName = $domain.Name
         $domainDN = $domain.GetDirectoryEntry().distinguishedName
         $adminSdHolderDN = "CN=AdminSDHolder,CN=System,$domainDN"
-
-        Write-Verbose "Working on Domain: $domainName"
-        Write-Verbose "DomainDN: $domainDN"
-
-        try {
-            $domainAcl = Get-ActiveDirectoryAcl $domainDN.ToString()
-            $adminSdHolderAcl = Get-ActiveDirectoryAcl $adminSdHolderDN
-
-            foreach ($group in $groupLists) {
-
-                Write-Verbose "Looking Ace Entries for the group: $($group.Name)"
-                foreach ($entry in $group.AceEntry) {
-
-                    if ((($entry.RootOnly) -and ($domainName -eq $rootDomainName)) -or
-                        ($entry.RootOnly -eq $false)) {
-                        Write-Verbose "Trying to find the entry GUID: $($entry.ObjectTypeGuid)"
-                        if ($entry.AdminSdHolder) {
-                            $objectAcl = $adminSdHolderAcl
-                            $objectDN = $adminSdHolderDN
+        $prepareDomainInfo = Get-ExchangeDomainConfigVersion -Domain $domainName
+
+        if ($null -ne $prepareDomainInfo.ObjectVersion) {
```

Comment:
This is not how to test if the domain ever ran `/preparedomain` the MESO container should only be in the root.

Reply:
Or is that just security groups in the root?? I need to look at this again.

Reply:
![image](https://user-images.githubusercontent.com/4518572/167859670-84d675ec-91a3-4289-a17f-43f58fa984bc.png)

Reply:
MESO should be available in every domain where /preparedomain has been run.

Reply:
> Or is that just security groups in the root?? I need to look at this again.

Yes, the `Microsoft Exchange Security Groups` are available only within the root domain.

Reply:
As discussed with @dpaulson45 we should be good by doing it this way. Resolving the conversation.